vpr/src/route/rr_graph.h

```diff
@@ -41,7 +41,7 @@ void create_rr_graph(const t_graph_type graph_type,
 void free_rr_graph();
 
 //Returns a brief one-line summary of an RR node
-std::string describe_rr_node(int inode);
+std::string describe_rr_node(int inode, const DeviceContext& device_ctx);
```

Comment:
Detail the comments about what this function does. Please keep in mind, make the comments being compatible with Doxygen format.
For example, 

```
@brief This function generates text in string format which describes detailed attributes for a given routing resource node. The attributes are essential for developers to diagnose when they hit a bug. The print-out message is helpful to find mismatches or unexpected nodes between the built routing resource graph and architecture description.
- In general, it shows the basic information, such as coordinate, number of fan-in and fan-out.
- It also shows more details depending on the node type. For example, as for routing tracks, it shows the directionality, the wire segment that represents the wire as well as track index in a routing channel.
```